### PR TITLE
Documentation for Balboa Spa integration

### PR DESCRIPTION
**Pull request by garbled1**
## Proposed change

Documentation for Balboa Spa integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/59234 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
Please don't stack different parent PR in a single PR.

This PR is now blocked while a parent PR has already been merged, thus a conflict.

Please revert the last addition.

File: source/_integrations/balboa.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ha_iot_class: Local Push
+ha_domain: balboa
+ha_platforms:
+  - climate
+ha_codeowners:
+  - '@garbled1'
+ha_config_flow: true
+---
+
+This integration adds "local push" support for [Balboa](http://www.balboawatergroup.com/) Spa WiFi Modules to be used within Home Assistant.
```
Comment (frenck) [inline source/_integrations/balboa.markdown]:
```suggestion
This integration adds support for [Balboa](http://www.balboawatergroup.com/) Spa WiFi Modules to be used within Home Assistant.
```

File: source/_integrations/balboa.markdown [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+There is currently support for the following device types within Home Assistant:
+
+- Climate
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+host:
+  description: Hostname or IP address of your Balboa Spa Wifi Device, e.g., 192.168.1.58
```
Comment (frenck) [inline source/_integrations/balboa.markdown]:
```suggestion
  description: "Hostname or IP address of your Balboa Spa Wifi Device, e.g., `192.168.1.58`."
```

File: source/_integrations/balboa.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  required: true
+  type: string
+sync_time:
+  description: Sync the Spa's internal clock with Home Assistant daily
```
Comment (frenck) [inline source/_integrations/balboa.markdown:34]: This is not part of the config flow.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/balboa.markdown (hunk 3 above)
Comment (garbled1) [reply source/_integrations/balboa.markdown:34]: Oh, sorry, thats part of the options flow.  Should that be documented in a different section?  I want to let the users know it's possible.

Comment (frenck) [reply source/_integrations/balboa.markdown:34]: That is possible, yes, the `cast` documentation has an example on this:

<https://www.home-assistant.io/integrations/cast#options>

**Review by frenck — APPROVED** [review]
Thanks, @garbled1 👍

**Pull request merged**